letsdrill.rb

```diff
@@ -1,24 +1,37 @@
+#letsdrill - Rajashree
 def get_letter_grade(integer)
- 
-  #Put your code here!
 
+  if integer >= 90 && integer <=100
+    return "A"
+  elsif integer >= 80 && integer < 90
+    return "B"
+  elsif integer >= 70 && integer < 80
+    return "C"
+  elsif integer >= 60 && integer < 70
+    return "D"
+  elsif integer < 60
+    return "F"
+  else
+    return "invalid"
+  end
 end
 
-def shortest_string(array)
+#puts get_letter_grade(105)
 
+def shortest_string(array)
   #Put your code here!
-
+  return array.min_by {|str| str.length }
```

Comment:
This is good. If you're just calling one method in a block, you can use the & (pretzel) operator. So here your it would look like
min_by &:length